### ISSUE TITLE
Morning session : peer to peer code review

### ISSUE THREAD

**Issue by muhammedb8826** [NicolasAndrehh/Awesome-Books#5]
You did a great job while building this project.
- The UI is perfect, it doesn't have an accessibility issue.
- Kindly make the navbar responsive.
